hoptimator-k8s/src/main/java/com/linkedin/hoptimator/k8s/K8sJobDeployer.java

```diff
@@ -41,6 +41,7 @@ public List<String> specify() throws SQLException {
         .with("database", job.sink().database())
         .with("schema", job.sink().schema())
         .with("table", job.sink().table())
+        .with("partialViewName", job.sink().partialViewName())
```

Comment:
Don't recall how `.with(x, null)` works. We may want to add `.withDefault(x, y, "")` or something like that.